Cargo.toml

```
@@ -32,32 +32,204 @@ exclude = [
     "forc-test/test_data"
 ]
 
+[workspace.package]
+edition = "2021"
+version = "0.63.5"
```

We're on a new version now :) 

```suggestion
version = "0.63.6"
```